package.json

```diff
@@ -72,6 +73,13 @@
     "gulp-clean": "0.4.0",
     "gulp-debug": "4.0.0",
     "gulp-typescript": "5.0.1",
+    "child_process": "1.0.2",
+    "cli-color": "2.0.0",
+    "delete-empty": "3.0.0",
+    "deprecate": "1.1.1",
+    "fs": "0.0.1-security",
+    "path": "0.12.7",
+    "util": "0.12.3",
```

Comment:
Uhm what is up with those dependencies? Were those meant to be here? :sweat_smile:

Reply:
The dependencies concern the functioning of the new Gulp script. 😀 Although now that I think about it I forgot to remove a `devDependencies` addiction. Which would be this:
`" deprecate ":" 1.1.1" `

as there is already above on `dependecies` 😔

do you want me to fix it or you?

Reply:
I am more concerned with `child_process`, `fs`, `path`, `util` - those are all part of Node.js standard library. Why do you want these as packages?

And yeah you can remove `deprecate` from the `devDependencies`.

`delete-empty` & `cli-color` is fine for me

Reply:
Yes, in fact `child_process`, `fs`, `path`, `util` I had to add them because it didn't work for me, but I was in doubt since as you said they are part of the standard Node.js library, now I have removed them with the last commit and it seems to .

I think it's the VSCODE editor that's gone crazy, even editors sometimes make fun of the developer 😂😂

However now it seems ok even as dependencies, another thing do you think about if we update the `typescript` version to`4.2.3` since I see that it is stuck at `4.0.3`?

Reply:
let me know if everything is ok in the `package.json`